Automattic/jetpack
# Beta plugin: Correcting testing tips links

Fixes #22104

#### Changes proposed in this Pull Request:

* In the 'To Test box', default text from the [testing tips page](https://github.com/Automattic/jetpack/blob/master/projects/plugins/beta/docs/testing/testing-tips.md) displays depending on the selection made. The 'Jetpack Beta Tester Plugin' link and the 'Jetpack Live Branches Readme' links have now been updated.
* Additionally, the line of text saying ‘The tips have been moved to the [Beta Plugin](https://karenattbeta.mystagingwebsite.com/projects/plugins/beta/docs/testing/testing-tips.md) in order that they can ship with them.’ showing in the 'To Test' box is located at jetpack/docs/testing/to-test.md (so is not produced by the Beta Plugin itself).
  * This box and text shows when clicking on ‘bleeding edge’ for the Beta Tester on a self-hosted Jetpack test site (I’ve not replicated it on a local development installation).

#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* From the Jetpack Beta Tester Home, click on the button to manage the Jetpack Beta Tester
* Make sure the 'existing version' is active (3.1.0)
* In the 'To Test' box, assuming the testing tips information is showing, test the links mentioned above.
